assets/js/components/HostDetails/HostDetails.jsx

```diff
@@ -79,38 +84,81 @@ function HostDetails() {
       />
       <div>
         <BackButton url="/hosts">Back to Hosts</BackButton>
-        <div className="flex">
-          <PageHeader>
-            Host Details: <span className="font-bold">{host.hostname}</span>
-          </PageHeader>
-          <StatusPill
-            className="self-center ml-4 shadow"
-            heartbeat={host.heartbeat}
-          >
-            Agent
-          </StatusPill>
-
-          {Object.entries(exportersStatus).map(
-            ([exporterName, exporterStatus]) => (
-              <StatusPill
-                key={exporterName}
-                className="self-center ml-4 shadow"
-                heartbeat={exporterStatus}
+        <div className="flex flex-wrap">
+          <div className="flex w-1/2 h-auto overflow-hidden overflow-ellipsis break-words">
+            <PageHeader>
+              Host Details: <span className="font-bold">{host.hostname}</span>
+            </PageHeader>
+          </div>
+          <div className="flex w-1/2 justify-end">
+            <div className="flex w-fit whitespace-nowrap">
+              {host.deregisterable && (
+                <CleanUpButton
+                  cleaning={host.deregistering}
+                  onClick={() => {
+                    setCleanUpModalOpen(true);
+                  }}
+                />
+              )}
+              <Button
+                type="primary-white"
+                className="inline-block mx-0.5 border-green-500 border"
+                size="small"
+                onClick={() => navigate(`/hosts/${hostID}/settings`)}
+              >
+                <EOS_SETTINGS className="inline-block fill-jungle-green-500" />{' '}
+                Check Selection
+              </Button>
+
+              <Button
+                type="primary-white"
+                className="mx-0.5 border-green-500 border disabled:bg-slate-50 disabled:text-slate-500 disabled:border-gray-400"
+                size="small"
+                onClick={() => navigate(`/hosts/${hostID}/executions/last`)}
+                disabled
+              >
+                <EOS_CLEAR_ALL
+                  className={classNames('inline-block ', {
+                    'fill-jungle-green-500': false,
+                    'fill-slate-500': true,
+                  })}
+                />{' '}
```

Comment:
Hey any reason for keeping the {' '} empty spaces? :D

Reply:
That's how the linter does it :see_no_evil: